### PR TITLE
CAD-1262: Disable scroll buttons on mobile

### PR DESCRIPTION
**Pull request by dblarons**

### CODE REVIEW

File: src/directives/mapbox.directive.js [hunk 1]
```diff
@@ -10,15 +10,16 @@
       scope: true,
       replace: true,
       link: function(scope, element, attrs) {
-        scope.map = L.mapbox.map(element[0], attrs.mapId);
+        var zoomControl = attrs.zoomControl === "false" ? false : true;
```
Comment (iwz) [inline src/directives/mapbox.directive.js]: can this just be `var zoomControl = attrs.zoomControl !== "false"`?

**Comment by aladata** [comment]
👍

File: src/directives/mapbox.directive.js (hunk 1 above)
Comment (dblarons) [inline src/directives/mapbox.directive.js]: Professor Roth would not be proud of me.

Comment (ThePumpingLemma) [inline src/directives/mapbox.directive.js]: bad, @dblarons, bad

File: src/directives/mapbox.directive.js [hunk 2]
```diff
@@ -31,6 +32,11 @@
         if (attrs.scrollWheelZoom === "false") {
           scope.map.scrollWheelZoom.disable();
         }
+        if (attrs.tap === "false") {
+          if (scope.map.tap) {
+            scope.map.tap.disable();
```
Comment (iwz) [inline src/directives/mapbox.directive.js]: this could probably be shortened up a bit:

`scope.map.tap && scope.map.tap.disable();`

Comment (dblarons) [inline src/directives/mapbox.directive.js]: I'm not a big fan of having actions in `&&` statements

Comment (iwz) [inline src/directives/mapbox.directive.js]: what about combing those nested `if`s then?

Comment (dblarons) [inline src/directives/mapbox.directive.js]: good point :) Will follow up with another PR

**Pull request merged**